tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -9680,18 +9697,34 @@ TEST_F(ExecutionTest, QuadAnyAll) {
     }
 
     if (!DoesDeviceSupportWaveOps(pDevice)) {
-      LogCommentFmt(L"Device does not support wave operations in shader model 6.%1u", ((UINT)sm & 0x0f));
+      LogCommentFmt(
+          L"Device does not support wave operations in shader model 6.%1u",
+          ((UINT)sm & 0x0f));
       continue;
     }
     Skipped = false;
 
     // test compute
-    std::shared_ptr<ShaderOpTestResult> test = RunShaderOpTestAfterParse(pDevice, m_support, "QuadAnyAll",
-      CleanUAVBuffer0Buffer, ShaderOpSet);
+    std::shared_ptr<ShaderOpTestResult> test = RunShaderOpTestAfterParse(
+        pDevice, m_support, "QuadAnyAll", CleanUAVBuffer0Buffer, ShaderOpSet);
 
     MappedData uavData;
     test->Test->GetReadBackData("UAVBuffer0", &uavData);
-    bool Result = VerifyQuadAnyAllResults((int2*)uavData.data());
+    bool Result = VerifyQuadAnyAllResults((int2 *)uavData.data());
+    VERIFY_IS_TRUE(Result);
```

Comment:
When these fail, it's not easy to tell which thing failed.  I think we need to have some more detail here if possible.  Perhaps this could be a future improvement though.